content/hectorqin/drone-kubectl/index.md

```diff
@@ -0,0 +1,185 @@
+---
+version: v0.1.0
+date: 2018-10-29T00:00:00+00:00
+title: Drone Kubectl
+author: hectorqin
+tags: [ deploy, kubernetes, docker ]
+logo: kubernetes.svg
+repo: hectorqin/drone-kubectl
+image: quay.io/hectorqin/drone-kubectl
+---
+
+# Kubernetes plugin for drone.io[![Docker Repository on Quay](https://quay.io/repository/hectorqin/drone-kubectl/status "Docker Repository on Quay")](https://quay.io/repository/hectorqin/drone-kubectl)
+
+This plugin allows to update all Kubernetes resources, not only the deployment but anything within a yaml template file by using kubectl command.
+
+## Usage
+
+This pipeline will update the resources of Kubernetes in the yaml file, which is generated from the template yaml file by bash, with `kubectl apply -f ./drone-k8s-gen.yaml` command.
+
+In the template yaml file, you can just use variable substitution to replace whatever you want, you can even add your own template helper, and use it in the template.
+
+
+```yaml
+    pipeline:
```

Comment:
You should provide Drone 1.x.x compatible examples.

Reply:
Sorry. I didn't see any example.

Reply:
Got it